napari/util/event.py

```diff
@@ -502,7 +503,7 @@ def __call__(self, *args, **kwargs):
     def _invoke_callback(self, cb, event):
         try:
             cb(event)
-        except Exception:
+        except Exception as exception:
```

Comment:
?

Reply:
I have no idea about this one. I can revert